app/src/org/commcare/android/fragments/SetupInstallFragment.java

```diff
@@ -54,6 +55,10 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 SetupEnterURLFragment enterUrl = new SetupEnterURLFragment();
+                Activity currentActivity = getActivity();
+                if (currentActivity instanceof CommCareSetupActivity) {
+                    ((CommCareSetupActivity)currentActivity).setUiState(CommCareSetupActivity.UiState.IN_URL_ENTRY);
+                }
```

Comment:
Not the cleanest fix, but this can be fixed in future refactors, which I will do when for the background auto-update feature I'm working on.